streampipes-extensions/streampipes-connect-adapters-iiot/src/main/java/org/apache/streampipes/connect/iiot/protocol/stream/FileReplayAdapter.java

```diff
@@ -133,66 +155,113 @@ public void onAdapterStarted(IAdapterParameterExtractor extractor,
         .getEventProperties()
         .stream()
         .filter(eventProperty ->
-            eventProperty.getDomainProperties().contains(URI.create(SO.DATE_TIME)))
+            eventProperty.getDomainProperties()
+                .contains(URI.create(SO.DATE_TIME)))
         .findFirst();
 
     if (timestampField.isEmpty()) {
       throw new AdapterException("Could not find a timestamp field in event schema");
     } else {
-      timestampRuntimeName = timestampField.get().getRuntimeName();
+      timestampRuntimeName = timestampField.get()
+          .getRuntimeName();
+    }
+
+    // check for renaming rules that affect the timestamp property
```

Comment:
Can you extract this functionality into a method? That could clean up the code a bit.